tag-expressions/python/Makefile

```diff
@@ -0,0 +1,92 @@
+# =============================================================================
+# GNU MAKEFILE: cucumber-tag-expressions for Python
+# =============================================================================
+# NOTE: bootstrap requires ON-LINE access.
+# REQUIRES: Python >= 2.7 is installed
+# REQUIRES: pip is installed (normally w/ newer python versions)
+# DESCRIPTION:
+#    This makefile uses "pipenv" to automatically setup a virtual environment
+#    (for Python) in the HOME directory of the user (for this directory).
+#
+# SEE ALSO:
+#  * https://pypi.python.org/pypi/pipenv/
+#  * https://docs.pipenv.org
+#  * https://github.com/pypa/pipenv
+# =============================================================================
+
+.PHONY: clean clean-all default test test.coverage tox
+
+# -----------------------------------------------------------------------------
+# PROJECT CONFIGURATION:
+# -----------------------------------------------------------------------------
+BOOTSTRAP_DONE_MARKER_FILE := build/.bootstrap.DONE
+BOOTSTRAP_DONE_MARKER_DIR  := $(dir $(BOOTSTRAP_DONE_MARKER_FILE))
+PIPENV := pipenv
+PIPENV_RUN := pipenv run
+PIPENV_VERSION := 8.3.2
+USE_PIPENV := yes
+SUDO := sudo
```

Comment:
The makefile use no "sudo" on Travis, if this is what is meant what you meant.
The makefile checks all assumptions and needs to install a python module "pipenv" if it is not installed yet. I had initially a simpler solution but it was not working on Travis or otherwise not working in a normal user workspace.

NOTE: Look at the complete "Makefile". If you have any questions, we can talk about it.